innodb-java-reader-cli/src/test/java/com/alibaba/innodb/java/reader/CsvPrinterTest.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Would you remove the header? Because currently we do not have license header in `src/test/java`. Probably I will add them all in the future.

Reply:
No problem.